software-templates/migrate-repo-to-enterprise/template.yaml

```diff
@@ -0,0 +1,300 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: GitHub-Repo-Transfer
+  title: Transfer a repository between organizations
+  description: Transfer a repository from the bcgov GitHub organization to the bcgov-public GitHub enterprise organization
+spec:
+  owner: "developer-experience"
+  type: service
+
+  parameters:
+    - title: Transfer a repository
+      required:
+        - repoToTransfer
+        - owner
+        - destination
+      properties:
+        repoToTransfer:
+          title: Repository to transfer
+          type: string
+          ui:field: RepoUrlPicker
+          ui:options:
+            requestUserCredentials: # this is super important - it causes the user to be prompted to log in to GitHub and/or authorize the required GH scopes.
+              secretsKey: USER_OAUTH_TOKEN
+              additionalScopes:
+                github:
+                  - workflow
+            allowedHosts:
+              - github.com
+            allowedOwners:
+              - bcgov
+        destination:
+          title: Destination GitHub Organization
+          type: string
+          description: GitHub organization where the repo will be transferred to.
```

Comment:
I suggest we default/pre-select `bcgov,-public` since it's the only option.

Reply:
Defaulted to `bcgov-public` with latest commit